vcd/provider.go

```diff
@@ -80,6 +82,12 @@ func Provider() terraform.ResourceProvider {
 				DefaultFunc: schema.EnvDefaultFunc("VCD_API_LOGGING_FILE", "go-vcloud-director.log"),
 				Description: "Defines the full name of the logging file for API calls (requires 'logging')",
 			},
+			"import_separation_token": &schema.Schema{
```

Comment:
Very fancy name for what it is, especially given that "token" is used so much in the security industry. Suggestion:
`import_separator_string`

Reply:
Changed to `import_separator`